pom.xml

```diff
@@ -105,7 +105,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>1.22</version>
+            <version>2.1.8</version>
```

Comment:
You shouldn't require the latest if there is no reason in APIs. Git 3.0 was so cool that i was forced to revert it to 2.x and hopefully nobody enforced latest version. So revert to latest possible according to APIs.